src/queries/topSymbols.ts

```diff
@@ -1,7 +1,10 @@
 import { gql } from "@apollo/client";
 
 export const usEquities = gql`
-  query SecuritiesQuery($input: SecurityFilterInput, $aggregatesInput: SecurityAggregateFilterInput) {
+  query SecuritiesQuery(
```

Comment:
These changes are only because my ide is configured to run prettier every time a file is touched  🥴 sorry for the noise

Reply:
you're good! I also have the prettier extension on my vscode